backend/src/main/java/ai/verta/modeldb/entities/metadata/MetadataParameterMappingEntity.java

```diff
@@ -0,0 +1,85 @@
+package ai.verta.modeldb.entities.metadata;
+
+import ai.verta.modeldb.metadata.IdentificationType;
+import ai.verta.modeldb.metadata.VersioningCompositeIdentifier;
+import ai.verta.modeldb.utils.ModelDBUtils;
+import java.io.Serializable;
+import java.util.Objects;
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
+import javax.persistence.EmbeddedId;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "metadata_parameter_mapping")
+public class MetadataParameterMappingEntity {
```

Comment:
I think Property would be a better name than Parameter

Reply:
replaced